docs/protocol/src/concepts/transactions.md

```diff
@@ -61,7 +61,7 @@ description leaves the value balance unchanged.
 transaction's value balance, burning them, and producing a swap commitment for
 use in the second stage;
 
-- **Sweep** descriptions perform the second phase of
+- **SwapClaim** descriptions perform the second phase of
```

Comment:
Reviewers: please confirm that this change is correct.

Reply:
Yep! Another terminology change we didn't update